lib/stormpath-sdk.rb

```diff
@@ -60,6 +60,8 @@ module Resource
     autoload :PasswordPolicy, 'stormpath-sdk/resource/password_policy'
     autoload :EmailTemplate, 'stormpath-sdk/resource/email_template'
     autoload :PasswordStrength, 'stormpath-sdk/resource/password_strength'
+    autoload :AccountSchema, 'stormpath-sdk/resource/account_schema'
```

Comment:
This should probably be called Schema, not Account Schema. The Account Schema should be the name of the association.